pkg/controller/kogitoapp/kogitoapp_controller_test.go

```diff
@@ -469,8 +469,32 @@ func TestReconcileKogitoApp_updateKogitoAppStatus(t *testing.T) {
 
 func TestReconcileKogitoApp_PersistenceEnabledWithInfra(t *testing.T) {
 	kogitoApp := createFakeKogitoApp()
+	kogitoApp.Spec.EnablePersistence = true
+	kogitoApp.Spec.EnableEvents = true
 	imgs := createFakeImages(kogitoApp.Name, map[string]string{framework.LabelKeyOrgKiePersistenceRequired: "true"})
-	fakeClient := test.CreateFakeClient([]runtime.Object{kogitoApp}, imgs, nil)
+	kogitoInfra := &v1alpha1.KogitoInfra{
```

Comment:
Wouldn't it have sense to create another test making sure that KogitoInfra is created when reconciliation is called and persistence and/or events are enabled?